com.dynamo.cr/com.dynamo.cr.bob.test/src/com/dynamo/bob/pipeline/ShaderProgramBuilderTest.java

```diff
@@ -783,7 +783,7 @@ public void testNativeReflectionAPI() throws Exception {
         byte[] spvSimple = getFile("simple.spv");
         assertNotNull(spvSimple);
 
-        long ctx = ShadercJni.NewShaderContext(spvSimple);
+        long ctx = ShadercJni.NewShaderContext(Shaderc.ShaderStage.SHADER_STAGE_FRAGMENT.getValue(), spvSimple);
```

Comment:
The "stage" is moved from the crosscompile function into creating the context constructor. This makes more sense as we already know upfront which stage it should be compatible with.